packages/tools/devtools/devtools-view/api-report/devtools-view.api.md

```diff
@@ -17,4 +20,11 @@ export interface DevtoolsPanelProps {
 
 export { IMessageRelay }
 
+// @public
```

Comment:
Should we make this `@internal`? I don't think we really intend for others to implement their own visualization at this stage.

Reply:
Yes, we should. Good call. I'll update this.